Source/Base/ASBaseDefines.h

```diff
@@ -18,6 +18,7 @@
 #import <Foundation/NSObjCRuntime.h>
 
 #define AS_EXTERN FOUNDATION_EXTERN
+#define unowned __unsafe_unretained
```

Comment:
Not sure if we should just be consistent and use `__unsafe_unretained` in a framework.

Reply:
Pros:
- Less underscores
- Shorter, cleaner

Cons:
- Not standard
- Could one day be a collision and we'd have to revert